CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ cmake_minimum_required (VERSION 3.1)
 
 SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake")
 
+cmake_policy(SET CMP0048 OLD) # project() doesn't have VERSION
```

Comment:
CMP0057 fix is done in #394.

Why CMP0048 here?

Reply:
I got an error of while running cmakefile due to the old polycies 
------
IN_LIST will be interpreted as an operator when the policy is set to NEW.
  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.19.2/share/cmake/Modules/CMakeFindDependencyMacro.cmake:47 (find_package)
  /Users/hngo/Documents/Codes/Libs/DGtal/DGtal/build/DGtalConfig.cmake:17 (find_dependency)
  CMakeLists.txt:18 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.
------